src/Web/Telegram/API/Bot/Data.hs

```diff
@@ -105,8 +106,13 @@ instance FromJSON Contact where
 
 -- | This object represents a chat.
 data Chat = Chat
-  {
-    chat_id :: Int                -- ^ Unique identifier for this chat, not exceeding 1e13 by absolute value
+  { chat_id :: Int64
```

Comment:
Docs https://core.telegram.org/bots/api#chat say here must be a number, not integer.
_ChatId_ is for fields of type _"Integer or String"_.

_Int64_ should be more efficient than _Integer_. Docs say it's safe to use _Int64_.

I'm not sure if it's safe to use _Int64_ in _ChatId_.